library/src/test/java/com/novoda/downloadmanager/NetworkRequestCreatorTest.java

```diff
@@ -0,0 +1,71 @@
+package com.novoda.downloadmanager;
+
+import org.junit.Test;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.novoda.downloadmanager.NetworkRequestFixtures.aNetworkRequest;
+
+public class NetworkRequestCreatorTest {
+
+    private final NetworkRequestCreator networkRequestCreator = new NetworkRequestCreator();
+
+    @Test
+    public void givenSimpleRequest_whenCreatingFileSizeHeadRequest_thenReturnsExpectedResumeRequest() {
```

Comment:
```suggestion
    public void createsFileSizeHeadRequest() {
```

I prefer the tests to be less verbose but 🤷‍♂️